notebooks/adversarial_search_fall_2021/.ipynb_checkpoints/index-checkpoint.ipynb

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
seems like your notebook isn't complete, I'll write my feedbacks based on the current version.